### ISSUE TITLE
IFCEngine error exception on import ifc file

### ISSUE THREAD

**Issue by maidi29** [helpsterTee/Unity-IFCEngine#15]
When trying to import the ifc file I get the following excpetion:

ifcengine assembly:<unknown assembly> type:<unknown type> member:(null)
  at (wrapper managed-to-native) IfcEngine.x86_64.sdaiOpenModelBN(int,string,string)
  
![image](https://github.com/helpsterTee/Unity-IFCEngine/assets/8269208/8e72acc9-c5be-4f58-815d-806afa5e7e97)

I did no changes to any script, just opened the sample scene and the manager script was called with the included ifc file

**Comment by helpsterTee** [comment]
Yeah this project is a bit older and I'm not working with Unity at the moment. 

My guess would be that they changed something in the IFC engine DLL since the last release. 

You can find older versions at https://rdf.bg/downloads-all/ifc-engine-downloads/

The last commit was from 2019 so it should work with one in this year. Can you try using an older version and see if the problem disappears?
